main.cpp

```diff
@@ -0,0 +1,228 @@
+// Configuration editor
+// Command line utiltiy to create configuration files
+// Valid arguments are init and edit
+
+#include <iostream>
+#include <string>
+#include <fstream>
+#include <stdio.h>
+
+
+using std::cout;
+using std::endl;
+using std::string;
+using std::cin;
+
+const string CONFIG_FILE_PATH = "/users/bconway/SourceTree/School/CPT-180-27-Assignment-2-Config-Generator/app.cfg";
+
+
+int main(int argc, char *argv[]) {
+
+    string argument1 = "";
+    // Make sure that the argument isn't empty
+    if (argv[1] != nullptr) {
```

Comment:
@bconway02 oh I'm sorry, I must have missed that! I will definitely go back through and check that out. I really like your use of the backup file. That was a great way of handling all the reading/writing :)